data/src/main/java/com/anwera97/data/di.kt

```diff
@@ -0,0 +1,23 @@
+package com.anwera97.data
+
+import com.anwera97.data.repository.CompanionRepository
+import com.anwera97.data.repository.ExpenditureRepository
+import com.anwera97.data.repository.TripRepository
+import org.koin.android.ext.koin.androidContext
+import org.koin.dsl.module
+
+val dataModule = module {
```

Comment:
consider migrate this to hilt, it is la leche

Reply:
Lets add it as a ticket